### PR TITLE
chore: Husky 🐶 

### PR DESCRIPTION
**Pull request by frassinier**
**What is the problem this PR is trying to solve?**

Prettier changes were not committed anymore,
Due to the list of changed files not being generated.

**What is the chosen solution to this problem?**

Use [Husky](https://github.com/typicode/husky) to be sure that matches our guidelines, before pushing anything.
Keep Prettier (and ESlint) GH Action for generated Pull Request from outside, even if it means losing all your Github Checks.

**Please check if the PR fulfills these requirements**

- [ ] The PR have used `yarn changeset` to a request a release from the CI if wanted.
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

**Review by jmfrancois — COMMENTED** [review]
please fill the description 
i do not understadn the motivation of the change.

File: .github/workflows/pr-lint.yml [hunk 1]
```diff
@@ -48,19 +51,29 @@ jobs:
         with:
           repo-token: "${{ secrets.GITHUB_TOKEN }}"
           report-json: ./eslint-report.json
+    
+      - name: Commit files
```
Comment (frassinier) [inline .github/workflows/pr-lint.yml:55]: If any!

File: .github/workflows/pr-lint.yml [hunk 2]
```diff
@@ -48,19 +51,29 @@ jobs:
...
-      - name: Commit prettier
-        uses: actions-js/push@master
-        if: steps.prettier.outputs.STATUS
+          for file in ${{ steps.changed-files.outputs.all_changed_files }}; do
+            if [[ $file == *.js || $file == *.ts || $file == *.jsx || $file == *.tsx || $file == *.mdx || $file == *.scss || $file == *.json ]]; then
+              echo "Run Prettier on $file"
+              npx prettier --config .prettierrc $file --write
+            fi
+          done
+          git commit -m "chore: prettier" ${{ steps.changed-files.outputs.all_changed_files }}
```
Comment (jsomsanith-tlnd) [inline .github/workflows/pr-lint.yml:71]: There is no `git add`

Comment (jmfrancois) [reply .github/workflows/pr-lint.yml:71]: add `-a` option should be enough

File: .github/workflows/pr-lint.yml [hunk 3]
```diff
@@ -48,19 +51,29 @@ jobs:
...
-          STATUS=$(git status --short output)
-          echo "::set-output name=STATUS::$STATUS"
-
-      - name: Commit prettier
-        uses: actions-js/push@master
-        if: steps.prettier.outputs.STATUS
+          for file in ${{ steps.changed-files.outputs.all_changed_files }}; do
+            if [[ $file == *.js || $file == *.ts || $file == *.jsx || $file == *.tsx || $file == *.mdx || $file == *.scss || $file == *.json ]]; then
+              echo "Run Prettier on $file"
+              npx prettier --config .prettierrc $file --write
```
Comment (jmfrancois) [inline .github/workflows/pr-lint.yml:68]: npx is super slow, please add it to the root package.json devdependencies

Comment (frassinier) [reply .github/workflows/pr-lint.yml:68]: Your rules! Will do!

**Review by jmfrancois — APPROVED** [review]

**Pull request merged**